cmd/tether/ops_linux.go

```diff
@@ -129,3 +133,105 @@ func (t *operations) Setup(sink tether.Config) error {
 
 	return nil
 }
+
+// SetupFirewall sets up firewall rules on the external scope only.  Any
+// portmaps are honored as are port exposes.
+func (t *operations) SetupFirewall(config *tether.ExecutorConfig) error {
+	// XXX It looks like we'd want to collect the errors here, but we
+	// can't.  Since this is running inside init (tether) and tether
+	// reaps all children, the os.exec package won't be able to collect
+	// the error code in time before the reaper does.  The exec package
+	// calls wait and attempts to collect its child, but the reaper will
+	// have raptured the pid before that.  So, best effort, just keep going.
+	_ = netfilter.Flush(context.Background(), "")
+
+	// default rule set
+	established := &netfilter.Rule{
+		Chain:  netfilter.Input,
+		States: []netfilter.State{netfilter.Established},
+		Target: netfilter.Accept,
+	}
+
+	reject := &netfilter.Rule{
+		Chain:  netfilter.Input,
+		Target: netfilter.Reject,
+	}
+
+	for _, endpoint := range config.Networks {
+		if endpoint.Network.Type == "external" {
```

Comment:
Should use the constant defined in the `github.com/vmware/vic/lib/portlayer/constants` package.